### ISSUE TITLE
After moving frame to a new location (with the mouse), the ivy-posframe still shows up in the old location on the screen

### ISSUE THREAD

**Issue by mtwomey** [tumashu/ivy-posframe#115]
As described in the title - I'm on emacs28, mac, [head recipe](https://github.com/daviderestivo/homebrew-emacs-head).

It looks like this after moving the frame:

![image](https://user-images.githubusercontent.com/7715262/140691795-ce0ce896-ef15-4937-95e1-e209f83c2c10.png)

It's (partially) outside the emacs frame / window and still in the same location it was in before moving the frame.

I'm unclear on the proper way to fix this, or even if it's an ivy-posframe issue (rather than a straight posframe issue). I was able to work around this problem with after advice as follows:

```
  ;; Hack to workaround how the posframe doesn't show up in the right place after moving the frame with the mouse
  (advice-add 'ivy-posframe-cleanup :after (lambda (&rest _)
    (posframe-delete ivy-posframe-buffer)))
```

Thoughts on a proper fix?

**Comment by tumashu** [comment]
maybe it is the bug of child-frame feature in mac emacs ：)

**Comment by mtwomey** [comment]
Yes , maybe - it's hard to tell where the bug is. One thing that I'm unclear on how ivy-posframe is _supposed_ to handle things when the frame is moved on the screen. Can you clarify this part, e.g. how is the frame move handled?

**Comment by tumashu** [comment]
when frame move, posframe should move too, it is child-frame's feature.
